manual/python/index.md

```diff
@@ -87,9 +173,53 @@ root [2] TPython::Prompt()
 
 For a more complete description of the `TPython` interface, please check the reference guide for {% include ref class="TPython" %}.
 
-## JupyROOT: PyROOT for Jupyter notebooks
+## JupyROOT: (Py)ROOT for Jupyter notebooks
+
+[Jupyter notebooks](https://jupyter.org/) provide a web-based interface that can be used to do interactive computing with ROOT.
+
+### How to use it
+
+Two kernels (or language flavours) allow to run ROOT from Jupyter: *Python* (2 and 3) and *C++*. In order to use such kernels, there are mainly two options:
```

Comment:
Is this fully correct? Is a kernel in Jupyter speech a language (like python or cpp) or a specific instance like py2, py3 or the root cpp kernel. I think the latter? In case this is true, it's not 100% technically correct to say "Two kernels ..."

Reply:
Yes, I think defining a kernel as a language flavour is accurate:

https://jupyter.readthedocs.io/en/latest/projects/kernels.html

it is true that you see the message "starting kernel" in Jupyter, but it rather means "kernel process" in that case.